plugins/messagelayer/consensus_plugin.go

```diff
@@ -255,16 +258,19 @@ func OpinionGiverFunc() (givers []opinion.OpinionGiver, err error) {
 	opinionGivers := make([]opinion.OpinionGiver, 0)
 
 	consensusManaNodes, _, err := GetManaMap(mana.ConsensusMana)
-
+	if err != nil {
+		plugin.LogErrorf("Error retrieving consensus mana: %s", err)
+	}
 	for _, v := range Registry().NodesView() {
-		// double check to exclude self
-		if v.ID() == local.GetInstance().ID() {
+		// double check to exclude self and check if node has enough mana to consider its statement
+		if v.ID() == local.GetInstance().ID() || !checkEnoughMana(v.ID(), StatementParameters.ReadManaThreshold) {
```

Comment:
I dont think we should apply checkEnoughMana here.
a) our perception might be wrong
b) if the node has to didact this from its allowed rate the statement has legitimately passed the rate control
c) if we don't make didact statements from the allowed rate we might have to FPC-vote on whether a statement is ok to be added to the tangle ->circular problem?